tests/Fixtures/Symfony/Kernel.php

```diff
@@ -6,15 +6,15 @@
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\Config\Resource\FileResource;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\Routing\RouteCollectionBuilder;
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
```

Comment:
Does this work for all Symfony versions?

Reply:
From 4.X onward, yes I believe so. I figured since this is only used in the tests it would be OK to update it.